Kanban/QSFU_kanban.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+#==============================================================================
+
+from redminelib import Redmine
+import pdb
+import datetime
+import logging
+import json
+import statistics
+import configparser
+
+config = configparser.ConfigParser()
+config.read('config_kanban.ini')
+
+URL = config['API']['URL']
+KEY = config['API']['ApiKey']
+
+redmine = Redmine(URL, key=KEY)
+
+prjlist = ['suseqa', 'openqav3', 'openqatests']
+taglist = ['y', 'epic', 'saga', 'fate']
+cycleList = []
+leadList = []
+
+id_codes = {
+        'New' : '1',
+        'In Progress' : '2',
+        'Resolved' : '3',
+        'Feedback' : '4',
+        'Workable' : 12
```

Comment:
I'd be consistent here and either use strings OR ints and convert them in-place where needed either with str(int_var) or the other way around with int(str_var).

Reply:
After reading how you use this I'd think about swapping key/value too.
It would enable a developer to easily map the internal numbers to a "human readable name". E.g. in L#37 a swap would enable the following:

```python
return id_codes.get(jsn.get("old_value", {}), None) in ["Workable", "New"]
```

But this is _really just a matter of taste_ ;)

Reply:
I just missed the marks here ^^

Reply:
The approach to switch key:value looks awesome, but the I would loose the ability to replace those magic numbers in the code?